### PR TITLE
Clarify the hit test and capturing target for boundary events

### LINKED ISSUE #145: Need to clarify what triggers boundary events when releasing a capture
When a pointer capture is released from an element A while the pointer is over a different element B, we expect <code>pointerout</code>/<code>pointerleave</code> events for <code>A</code> and <code>pointerover</code>/<code>pointerenter</code> events for B before sending out future events (say a <code>pointermove</code>) to the hit-test node B. The current spec wording covers this through the Note in [Section 5.1.3](https://w3c.github.io/pointerevents/#firing-events-using-the-pointerevent-interface).

The wording needs to be a bit more precise, see [the discussion in a previous PR](https://github.com/w3c/pointerevents/pull/138#r77719958).

### PR DESCRIPTION
**Pull request by NavidZ**
closes #145 
I followed a few suggestions mentioned in the bug.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -634,7 +634,7 @@
...
+                    <p><a href="#setting-pointer-capture">Sets</a> <a>pointer capture</a> for the pointer identified by the argument <code>pointerId</code> to the element on which this method is invoked. For subsequent events of the pointer, capturing target will substitute the normal hit testing result as if the pointer is always over the capturing target and they MUST always be targeted at this element until capture is released. The pointer MUST be in its <a>active buttons state</a> for this method to be effective, otherwise it fails silently. Throws a <code>DOMException</code> with the name <code>InvalidPointerId</code> when the provided method's argument does not match any of the <a data-lt="active pointer">active pointers</a>.</p>
```
Comment (patrickhlauke) [inline index.html]: Suggest a slight rewording: "For subsequent events of the pointer, _the_ capturing target will substitute the normal hit testing result...target, and they MUST always.." (note the added comma after "target" too)

**Comment by patrickhlauke** [comment]
LGTM barring the slight tweaks to the last changed sentence.

**Comment by NavidZ** [comment]
Any other comments on this this? Is it okay to land this?

**Comment by teddink** [comment]
Looks good - I reviewed this with development last week,

**Comment by RByers** [comment]
LGTM

**Pull request merged**